src/database.providers.ts

```diff
@@ -1,9 +1,15 @@
 import { dbCredentialsString } from '@db/config';
+import * as usersSchema from '@users/users.model';
 import { type PostgresJsDatabase, drizzle } from 'drizzle-orm/postgres-js';
 import postgres from 'postgres';
 
 export const migrationsClient = postgres(dbCredentialsString, { max: 1 });
 
 export const queryClient = postgres(dbCredentialsString);
 
-export const db: PostgresJsDatabase = drizzle(queryClient);
+export const CombinedSchemas = { ...usersSchema };
```

Comment:
```suggestion
export const combinedSchemas = { ...usersSchema };
```
PascalCase should be reserved for types / classes / decorators